plugin/pkg/scheduler/scheduler.go

```diff
@@ -287,6 +288,10 @@ func (sched *Scheduler) bind(assumed *v1.Pod, b *v1.Binding) error {
 
 // scheduleOne does the entire scheduling workflow for a single pod.  It is serialized on the scheduling algorithm's host fitting.
 func (sched *Scheduler) scheduleOne() {
+	if !leaderelection.IsLeader() {
+		sched.StopEverything()
```

Comment:
This is a bad idea.  The hard exit exists due to concurrency.  Without fencing we need to exit(1) as fast as possible